### ISSUE TITLE
Bitfinex cannot be used with Binance due to conflicing versions of `ring`

### ISSUE THREAD

**Issue by AlexApps99** [wisespace-io/bitfinex-rs#10]
```
error: failed to select a version for `ring`.
    ... required by package `bitfinex v0.4.0`
    ... which is depended on by `testcrate v0.1.0 (/home/test/testcrate)`
versions that meet the requirements `^0.14` are: 0.14.6, 0.14.5, 0.14.4, 0.14.3, 0.14.2, 0.14.1, 0.14.0

the package `ring` links to the native library `ring-asm`, but it conflicts with a previous package which links to `ring-asm` as well:
package `ring v0.16.0`
    ... which is depended on by `binance v0.10.0`
    ... which is depended on by `testcrate v0.1.0 (/home/test/testcrate)`

failed to select a version for `ring` which could resolve this conflict
```

**Comment by AlexApps99** [comment]
I see that the version of `ring` in this repository is not conflicting, perhaps a patch version could be released with the changes to dependency versions applied

**Comment by wisespace-io** [comment]
@AlexApps99 I just tried to build with both libraries and I did not have any problem, see output below:

    $ cargo build --release
    
    ...
    Compiling binance v0.10.2 (https://github.com/wisespace-io/binance-rs.git#0f0e06b3)
    Compiling bitfinex v0.4.0 (https://github.com/wisespace-io/bitfinex-rs.git#106e87f2)
    Compiling crypto-test v0.1.0 (/home/oliveira/crypto-test)
    Finished release [optimized] target(s) in 4m 41s

    $ rustc --version
    rustc 1.42.0 (b8cedc004 2020-03-09)

Could you run a "cargo update" and try to build again?

**Comment by wisespace-io** [comment]
Okay, I can see the problem.

Quick fix is to update your Cargo.toml to use

    [dependencies]
    binance = "0.10.2"
    bitfinex = { git = "https://github.com/wisespace-io/bitfinex-rs.git" }

I never released a bitfinex version with the ring 0.16 version, I will do it later.
Thanks for reporting the issue.
